graf2d/gpad/src/TPad.cxx

```diff
@@ -2844,6 +2844,10 @@ TFrame *TPad::GetFrame()
       fFrame->SetLineWidth(GetFrameLineWidth());
       fFrame->SetBorderSize(GetFrameBorderSize());
       fFrame->SetBorderMode(GetFrameBorderMode());
+   } else {
+      // Preexisting and now assigned to fFrame, let's make sure it is not
+      // deleted twice (the bit might have been set in TPad::Streamer)
+      fFrame->ResetBit(kCanDelete);
```

Comment:
That is a good question.  I am not sure it is 'possible' (that `fFrame` was not null and set to a different value).  @couet do you know if there was/is a possibility of memory leak here (i.e. in which case could fFrame change to non-null to another non-null value)?

Reply:
Seems to me, as `fFrame` is internal to `TPad`, I do not see how it can point to something else than the value set by `GetFrame`. There is no `SetFrame` (unlike `SetHistogram` we have in many places in graphics primitives).